packages/mobile/src/screens/product-info/components/seller-info/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { FontFamily, FontSize } from '~/common/enums/enums';
+import { ColorPalette } from '@vse-bude/shared';
+import { StyleSheet } from 'react-native';
```

Comment:
Re-organize imports order according to common rules

Reply:
fix